js_modules/dagster-ui/packages/ui-core/src/assets/AutomaterializePolicyTag.tsx

```diff
@@ -14,7 +14,7 @@ export const AutomaterializePolicyTag: React.FC<{
 
 export const automaterializePolicyDescription = (policy: {
   policyType: AutoMaterializePolicyType;
-  rules: AutoMaterializeRule[];
+  rules: {description: string; decisionType: AutoMaterializeDecisionType}[];
```

Comment:
I would use `Pick<AutoMaterializeRule, 'description' | 'decisionType'>` but this is fine too

Reply:
Ahh I should do this more often, actually going to make a separate PR and see if I can change a bunch of these helpers to use Pick.

Reply:
Hmm build failed for the eslint upgrade so I went ahead and changed this here